tinder.js

```diff
@@ -63,9 +66,32 @@ function TinderClient() {
    * @param {Object} data an object containing extra values
    * @param {Function} callback the callback to invoke when the request completes
    */
-  var tinderPost = function(path, data, callback) {
-    var opts = getRequestOptions(path, data);
+  var tinderPost = function(path, data, callback, file) {
```

Comment:
I had to modify a little bit the metho tinderPost in order to add the send file support here